readme.md

```diff
@@ -40,6 +40,32 @@ console.log(obj);
 //=> {foo: {bar: 'b', baz: 'x', 'dot.dot': 'unicorn'}}
 ```
 
+
+## API
+
+### get(obj, path)
+
+### set(obj, path, value)
+
+#### obj
+
+Type: `object`
+
+Object to get or set the path value.
+
+#### path
+
+Type: `string`
+
+Path of the property in the object.
```

Comment:
@sindresorhus Do you mean it's ok as is, or do you want me to rephrase that with the dot property stuff?

Something like?

> Path of the property in the object. Use `.` for nested objects or `\\.` to add a `.` in a key.